src/Microsoft.AspNet.Mvc.Razor/Resources.resx

```diff
@@ -148,7 +148,7 @@
     <value>There is no active writing scope to end.</value>
   </data>
   <data name="RazorPage_YouCannotFlushWhileInAWritingScope" xml:space="preserve">
-    <value>You cannot flush while inside a writing scope.</value>
+    <value>You cannot flush while inside a writing scope. (Path: '{0}')</value>
```

Comment:
`invocation` sounds weird and so does `that method`... Though the original string said `you cannot flush` which while true, is irrelevant: people can never flush anything while inside a writing scope. It's more like `The XYZ operation cannot be performed in ABC mode in 'PATH'.` or something, no?